validator/testcases/javascript/predefinedentities.py

```diff
@@ -228,7 +228,7 @@ def is_shared_scope(traverser, right=None, node_right=None):
             '`InstallTrigger` API: '
             'https://developer.mozilla.org/en-US/docs/Web/API/InstallTrigger',
             'Updates to existing add-ons should be provided via the '
-            'install.rdf `updateURL` mechanism.'),
+            'manifest\'s `updateURL` mechanism.'),
```

Comment:
You mean changing all the simple quotes with double quotes in this instance? I can do that, sure. I'm not proposing to change only on this line, as it would look weird, right?